frontend/helpers/comments.ts

```diff
@@ -300,6 +308,30 @@ export const transformCustomsLinksToMarkdown = ( comment: IComment, tickets, typ
 	const usersReferences = text.matchAll(MARKDOWN_USER_REFERENCE_REGEX);
 	const ticketsReferences = text.matchAll(MARKDOWN_TICKET_REFERENCE_REGEX) || [];
 	const resourcesReferences = text.matchAll(MARKDOWN_RESOURCE_REFERENCE_REGEX);
+	const viewpointReferences = text.matchAll(INTERNAL_VIEWPOINT_ID_REGEX);
+
+	if (viewpointReferences) {
+		const teamspace = details.account;
+		const projectId = details.model;
+		const issueId = details._id;
+		let referenceType;
+
+		if (type === 'issue') {
+			referenceType = 'issues';
+		} else if (type === 'risk') {
+			referenceType = 'risks';
+		}
+
+		const uniqViewpointReferences = uniqBy([...viewpointReferences], 0);
+		uniqViewpointReferences.forEach(({0: viewpointReference}) => {
```

Comment:
`{ 0: viewpointReference }`
There should be additional spaces, it just our syntax convention, not an error.